templates/workspace_services/guacamole/guacamole-server/docker/version.txt

```diff
@@ -1 +1 @@
-__version__ = "0.9.4"
+__version__ = "0.9.5"
```

Comment:
I think you need to update the bundle version too, otherwise only new TRE deployments will catch this change.

Reply:
+1

Reply:
Done.